scripts/generate_archive_model.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I don't understand why we're calling these "Archive". When a new version (say _10) is released we will not be able to rename the current version - it's a Go import path. 

I think all of the camt.060 versions should go under a `gen/AccountReportingRequest/` folder.

Reply:
We can change the current directories - no one is using this library yet.